### PR TITLE
win_domain - Add ability to specify netbios name

### PR DESCRIPTION
**Pull request by jtauke**
##### SUMMARY

Changes are to win_domain.ps1 and win_domain.py.  The changes allow a user to specify the netbios name of the newly created domain.  This is helpful when following best practice from microsoft and specifying an internal domain for Active Directory (i.e. ad.test.com or int.test.com).  Prior to this change the netbios name of the domain would default to AD or INT using the examples above.  After this change, the netbios name would be TEST.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

win_domain.ps1
win_domain.py
##### ANSIBLE VERSION

```
ansible 2.5.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/joe.tauke/Dev/ansible/lib/ansible/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.14 (default, Mar 14 2018, 13:36:31) [GCC 7.3.1 20180303 (Red Hat 7.3.1-5)]

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by jtauke** [comment]
ready_for_review

File: lib/ansible/modules/windows/win_domain.ps1 [hunk 1]
```diff
@@ -80,7 +81,10 @@ If(-not $forest) {
         if ($sysvol_path) {
             $install_forest_args.SysvolPath = $sysvol_path
         }
-
+	if ($domain_netbios_name) {
+        $install_forest_args.DomainNetBiosName = $domain_netbios_name
```
Comment (jhawkesworth) [inline lib/ansible/modules/windows/win_domain.ps1]: I know this is super-fussy, but I would prefer it if this line was indented to match the other if blocks in the module.

**Comment by jhawkesworth** [comment]
If you could fix the indentation (see review comments above) I would be happy to support merging this in.

**Comment by jtauke** [comment]
Indentation should be properly set now.

**Comment by jhawkesworth** [comment]
Thanks for fixing the indentation.  Sorry I just thought of this, and I know its only a litle thing... but it would be good to describe what happens if the netbios name is not specified in the documentation.  Something like...  Default: same as the first part of the dns_domain_name, up to, but not including the '.'  (feel free to describe it better than this).

Otherwise I think this is ready to be merged.

File: lib/ansible/modules/windows/win_domain.py [hunk 2]
```diff
@@ -21,6 +21,11 @@
     description:
       - the DNS name of the domain which should exist and be reachable or reside on the target Windows host
     required: yes
+  domain_netbios_name:
+    description:
+      - the netbios name of the domain
+    required: no
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_domain.py]: `required: no` is the default and we usually just omit it.

File: lib/ansible/modules/windows/win_domain.py [hunk 3]
```diff
@@ -21,6 +21,11 @@
     description:
       - the DNS name of the domain which should exist and be reachable or reside on the target Windows host
     required: yes
+  domain_netbios_name:
+    description:
+      - the netbios name of the domain
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_domain.py]: I know the other parts of the documentation isn't in a sentence (we should fix this) but usually the description is capitalized and ends with `.`

**Review by jborean93 — COMMENTED** [review]
Just minor nitpicks, I agree that the description for the new option can be expanded a bit.

**Comment by jtauke** [comment]
@jborean93 and @jhawkesworth, thanks for the review comments. I just committed some alterations around the description for domain_netbios_name (as well as uniformity changes to bring other sections in line with sentence format).

**Review by jborean93 — APPROVED** [review]
Thanks @jtauke

**Pull request merged**